test/typescript/almin-test.ts

```diff
@@ -85,15 +85,15 @@ context.onDispatch((payload: Payload, meta: DispatcherPayloadMeta) => {
 });
 // UseCase
 class ChildUseCase extends UseCase {
-    execute(value: string) {
+    execute(value: string): any {
```

Comment:
The limitation of this approach.
UseCase#execute === UseCaseExecutor#execute on Typing.